### PR TITLE
Update storage generator to suppress FxCop issues

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

**Review by seanmcc-msft — APPROVED** [review]

**Comment by seanmcc-msft** [comment]
@pakrym, thank you!

**Comment by pakrym** [comment]
We can't merge this PR it contains breaking API changes because the currently checked in generated code seems to be manually modified.

**Comment by seanmcc-msft** [comment]
> We can't merge this PR it contains breaking API changes because the currently checked in generated code seems to have been manually modified.

That is odd, the code should not have been manually modified.  What happens if you regenerate?

**Comment by pakrym** [comment]
>What happens if you regenerate?

The code in this PR is the one I've got when running .\generate.ps1

**Comment by seanmcc-msft** [comment]
I am investigating.

**Comment by seanmcc-msft** [comment]
It looks like a lot of BlobFactory method parameters were reordered.  We didn't make this change in the 2020-02-10 swagger or readmes.  Has the generator been updated to have a different behavior around parameter order?

**Comment by pakrym** [comment]
Not as part of this PR.

**Comment by seanmcc-msft** [comment]
I mean like has the .NET generator be modified recently, including other PRs.

I am not able to repo this issue locally.  Off the master branch, I can generate and pass `eng\scripts.Export-API.ps1` after adding the pragma disables.

File: sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs [hunk 1]
```diff
@@ -16251,10 +16251,10 @@ public static partial class BlobsModelFactory
         /// Creates a new BlobBlock instance for mocking.
         /// </summary>
         public static BlobBlock BlobBlock(
-            string name,
-            int size)
+            int size,
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs:16254]: This is not occurring when I generate locally.

**Comment by seanmcc-msft** [comment]
Oh!  I know the problem.  @pakrym which version of node are you using?

The generator puts the parameters in the wrong order if you are not using= [v10.15.3](https://nodejs.org/dist/v10.15.3/)

**Pull request merged**